Luke Smith Portfolio.css

```diff
@@ -0,0 +1,111 @@
+/** {
+	box-sizing: border-box;
+}
+*/
+body {
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.topnav {
+  overflow: auto;
+  background-color: #ffffff;
+}
+
+.topnav a {
+  float: left;
+  color: #000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  width: auto;
+}
+
+.topnav a:hover {
+  background-color: #dddddd;
+  color: #000000;
+}
+
+body {
+  background: #555555;
+}
+
+
+/*About Me Section*/
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: black;
+  margin: 0;
+  max-width: 500px;
+  margin: auto;
+  padding: 10px;
+}
+
+.column {
+  float: left;
+  width: 33.3%;
+  margin-bottom: 16px;
+  padding: 0 8px;
+}
+
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  margin: auto;
+}
+
+.about-section {
+  padding: 50px;
+  text-align: center;
+ 
+}
+
+.container {
+  padding: 0 16px;
+}
+
+.container::after, .row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+.title {
+  color: black;
+}
+
+.button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 8px;
+  color: white;
+  background-color: #000;
+  text-align: center;
+  cursor: pointer;
+  width: 100%;
+}
+
+.button:hover {
+  background-color: #555;
+}
+
+
+/*Contact section links*/
+
+  .contact:link, .contact:visited {
```

Comment:
You don't need .contact:link, .contact:visited here. Just .contact will do - this will apply the styling rule to all the elements with a class of "content" while you are not hovering over them, which is exactly what your current styling rules are achieving.